About.html

```diff
@@ -0,0 +1,118 @@
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <header>
+  
+        <nav class="navbar">
+           <h1> <a href="./index.html"><span>Buganda</span> KingsMen</a></h1>
+            <ul>
+                <li><a href="#" >About</a></li>
+                <li><a href="#" >Program</a></li>
+                <li><a href="#" >Join</a></li>
+                <li><a href="#" >Sponser</a></li>
+                <li><a href="#" >New</a></li>
+                <li><button>CC Campaign</button></li>
+            </ul>
+       
+        </nav>
+</header>
+<div>
+    <div class="headding">
+        <div>.</div>
+      <ul class='Navmenu'>
+        <li class="nav-items"><a class="nav-links" href="./index.html">Home</a> </li>
+        <li class="nav-items"><a class="nav-links" href="./About.html">About</a> </li>
+        <li  class="nav-items"><a class="nav-links"  href="#about">Program</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Join</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Sponser</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">New</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">CC Campaign</a></li>
+      </ul>
+      <div class="hamburger-menu">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+
+    </div>
+</div>
+<section class="intro-sections">
+        <div class="intro-wrapper">
+            <h3 class="intro-wrappers-h3">"The Buganda Pride"</h3>
+            <h1>THE BUGANDA CULTURAL PROSPEROUS <br/>FUTURE FLEX  2022</h1>
+        </div>
+        <p>
+            Buganda is a Bantu kingdom within Uganda. The kingdom of the Baganda people, Buganda is the largest of the traditional kingdoms in present-day East Africa, consisting of Buganda's Central Region, including the Ugandan capital Kampala. The 14 million Baganda (singular Muganda; often referred to simply by the root word and adjective, Ganda) make up the largest Ugandan region, representing approximately 26.6% of Uganda's population.
+        </p>
+        <br>
+        <br>
+        <span>Please contact us via Email for any further questions about the Science Conference 2023</span>
+        <br>
+        <br>
+        <span>culture@bugand-kingdom.org</span>
+</section>
+<section class="intro-sectioned">
+    <h3 class="intro-sectioned-h3">The Buganda KingsMen cultural Galla</h3>
+    <p>
+    The logo of the Global submit was decided through the logo competition form 8th June to 7th July. After online votiong and election commitee's judging process, the logo from Nanjal Gumbson won the price.
+     After consulting Nanjal Gumbson, we optimized the prize winning logo for this years as seen below.
+    </p>
+    <h1 class="kingsman"><span class="uganda">BUGANDA</span> KINGSMEN</h1>
```

Comment:
Kindly, I suggest applying only 1 H1 header on your site, followed by H2, H3... in order of importance, this is good HTML practice and will help a lot with the accessibility of your site.